clnclient.py

```diff
@@ -71,10 +71,10 @@ def refresh(self):
                 chan.local_base_fee, chan.local_fee_rate = fee_local
                 chan.remote_base_fee, chan.remote_fee_rate = fee_remote
                 chan.local_alias = self.local_alias
-                chan.remote_alias = self._run("listnodes", chan.remote_node_id)[
-                    "nodes"
-                ][0].get("alias")
-                if not chan.remote_alias:
+                listnode = self._run("listnodes", chan.remote_node_id)
+                if len(listnode["nodes"]) > 0:
+                        chan.remote_alias = listnode["nodes"][0].get("alias")
```

Comment:
Let's also handle the case where "alias" is not set:

```suggestion
                        chan.remote_alias = listnode["nodes"][0].get("alias", chan.remote_node_id)
```